docs/charon/charon-cli-reference.md

```diff
@@ -163,7 +163,7 @@ Flags:
       --data-dir string                The directory where charon will store all its internal data (default ".charon")
       --definition-file string         The path to the cluster definition file or an HTTP URL. (default ".charon/cluster-definition.json")
   -h, --help                           Help for dkg
-      --keymanager-address string      The keymanager URL to push validator keyshares to.
+      --keymanager-address string      The keymanager URL to import validator keyshares.
```

Comment:
here also
```suggestion
      --keymanager-address string      The keymanager URL to export validator keyshares.
```

Reply:
The reason we use the verb "import" because the [keymanager API](https://ethereum.github.io/keymanager-APIs/#/Local%20Key%20Manager/importKeystores) uses the same wording. But, yeah we need to evaluate if it is easy to understand by the users

Reply:
Let's agree to use the words ("import") of the keymanager spec since it's safer and intuitive